content/posts/linux/Desktop Linux Hardening.md

```diff
@@ -247,7 +247,7 @@ firewall-cmd --add-service=dhcpv6-client --permanent
 firewall-cmd --reload
 ```
 
-On some distributions, it may be possible for unauthorized users or applications to make firewall changes through polkit. To disable this, enable firewalld _lockdown mode_ with `sudo firewall-cmd --lockdown-on`.
+On some distributions, it may be possible for applications running as a `wheel` or `sudo` user to make firewall changes through polkit. To disable this, enable firewalld _lockdown mode_ with `sudo firewall-cmd --lockdown-on`.
```

Comment:
```suggestion
On some distributions, it may be possible for applications running as a `wheel` or `sudo` user to make firewall changes through polkit. To prevent this, enable firewalld _lockdown mode_ with `sudo firewall-cmd --lockdown-on`.
```